### DISCUSSION TITLE
Beginning Questions

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [smplayer-dev/smplayer#431] [Q&A]
Hi there,

Is there a setting to prevent the player being sent to system tray after closing? (Ie, I'd like it to stop running altogether after closing).

Thanks in advance 🙂

Edit:

Also: I notice that after clicking on the "File (Ctrl+F)" button, I'm unable to select more than one file at a time...

Can anyone clarify which button/setting/keyboard shortcut to use in order to do so?

**Comment by smplayer-dev** [comment]
Turn off the option "show icon in system tray" in the "options" menu.
You can open several files at a time in the playlist (add button -> add files).

**Reply by GitAlexToTheGreek** [reply]
> Turn off the option "show icon in system tray" in the "options" menu. You can open several files at a time in the playlist (add button -> add files).

Hi smplayer-dev 👋

Thanks so much!

I'd like to 'mark as answer' but also keep the thread going for future questions; that ok?
